ERCS/erc-7879.md

```diff
@@ -0,0 +1,106 @@
+---
+EIP: 7879
+Title: Program-Derived Addresses (PDAs) 
+Author: John Crunch, @JohnCrunch
+Discussions-to: TBD
+Status: Draft
+Type: Standards Track
+Category: ERC
+Created: 2025-2-08
+---
+
+## Abstract
+This Ethereum Improvement Proposal (EIP) introduces Program-Derived Addresses (PDAs) as a mechanism to generate deterministic, contract-derived addresses without private keys. Inspired by Solana’s PDA architecture, this standard enables smart contracts to create verifiable and immutable addresses using predefined seeds and cryptographic operations. PDAs provide stateless, permissionless interactions with smart contracts while ensuring security and uniqueness.
```

Comment:
Your abstract is decent, but it's missing a high-level (but still technical) overview of _how_ the proposal accomplishes its goals. You should sketch out what the derivation mechanism is, at least briefly.